lib/standalone.mjs

```diff
@@ -124,6 +126,8 @@ export default async function standalone({
 				? join(cwd, codeFilename)
 				: codeFilename;
 
+		formatterFunction = await getFormatter(stylelint, { filePath: absoluteCodeFilename });
```

Comment:
[suggest] I think we should call `getFormatter()` at once. In other words, the `formatter` should be determined in a root-level config. If it's possible, the `filePath` parameter will be no longer necessary.

Reply:
Not sure I understand correctly... root-level instead of on a per-file basis? Maybe by using `resolveConfig()` instead of `getConfigForFile()` ?

Reply:
> root-level instead of on a per-file basis?

Yes.

> Maybe by using `resolveConfig()` instead of `getConfigForFile()` ?

No. I think it's unnecessary to pass the third `filePath` argument to `getConfigForFile()` if I understand correctly.

Reply:
Hmmm. I'm pretty sure at one point some tests were failing without the filePath argument... but, hey it seems to work now without it. Hehe. Not sure I understand why/how though, sadly.